src/visitor.rs

```diff
@@ -185,6 +185,29 @@ impl<'a, 'v> visit::Visitor<'v> for FmtVisitor<'a> {
         if self.visit_attrs(&ti.attrs) {
             return;
         }
+
+        self.format_missing_with_indent(ti.span.lo);
+
+        match ti.node {
```

Comment:
Could use an `if let` here rather than match (although we'll want a match eventually, so it doesn't matter too much)